packages/gatsby-plugin-sharp/README.md

```diff
@@ -85,6 +85,7 @@ plugin for Markdown content as gatsby-remark-images).
 * `maxWidth` (int, default: 800)
 * `maxHeight` (int)
 * `quality` (int, default: 50)
+* `accountForPixelDensity` (bool, default: true)
```

Comment:
How about `sizeByPixelDensity`. "account" is a bit ambiguous as to what exact action will be taken because of the option.

Also let's default to `false`. That's what we had before and there was only one person that wanted the default switched and since switching the default we've had ~6 or so people complain about their images being busted.